examples/kms_discovery.cpp

```diff
@@ -14,11 +14,15 @@
  */
 
 #include <aws/cryptosdk/cpp/kms_keyring.h>
-#include <aws/cryptosdk/default_cmm.h>
 #include <aws/cryptosdk/session.h>
 
-const char *KEY_ARN_US_WEST_2    = "arn:aws:kms:us-west-2:658956600833:key/b3537ef1-d8dc-4780-9f5a-55776cbb2f7f";
-const char *KEY_ARN_EU_CENTRAL_1 = "arn:aws:kms:eu-central-1:658956600833:key/75414c93-5285-4b57-99c9-30c1cf0a22c2";
+/* This example shows how various KMS keyrings operate slightly differently on
+ * decryption. The code is completely functional except for the two fake KMS
```

Comment:
Do we want to document the difference in operation that is being referred to here? It is not clear to me what the comment is trying to convey.

Reply:
The comment was trying to give an overall context of what this example is about. It's not a realistic one because no one needs to make six different keyrings to do the exact same decryption. It's meant to be illustrative of what the differences in behavior of those different keyrings are through the comments lower down in the file. I'll reword this to make it a little clearer.